### ISSUE TITLE
Issue with group constants

### ISSUE THREAD

**Issue by BryanBrasil** [uniVocity/univocity-html-parser-api#1]
Hi, thanks for the nice product! I've found a bug defining constants in groups.

If you create a group with a constant defined as the last field the values will be null:

```
    Group group = entity.newGroup().startAt("tr").withExactText("group start").endAt("tr").withExactText("group end");
    group.addField("product").match...;
    group.addField("price").match...;
    group.addField("a_constant", "value");
```

If I define "a_constant" before a field then it works.

**Comment by jbax** [comment]
Checking, thank you for reporting this bug!

**Comment by jbax** [comment]
Fixed and released version 2.0.1 with the adjustment
Thank you again!
